### PR TITLE
ref #54, #52: use HttpClient instead of WebClient

### PR DESCRIPTION
**Pull request by akornich**

### CODE REVIEW

File: Rollbar/RollbarResponse.cs [hunk 1]
```diff
@@ -27,6 +27,15 @@ public class RollbarResponse
...
         public RollbarResult Result { get; set; }
 
+        /// <summary>
+        /// Gets or sets the HTTP details.
+        /// </summary>
+        /// <value>
+        /// The HTTP details.
+        /// </value>
+        [JsonIgnore]
+        public string HttpDetails { get; set; }
```
Comment (rokob) [inline Rollbar/RollbarResponse.cs]: What is the point of this string? Is it just for debugging?

**Comment by rokob** [comment]
LGTM

File: Rollbar/RollbarResponse.cs (hunk 1 above)
Comment (akornich) [reply Rollbar/RollbarResponse.cs]: yes, it provides you with http request-response strings that were exchanged with the server.

**Comment by akornich** [comment]
@rokob, thanks for the review.

**Pull request merged**